app/status/views.py

```diff
@@ -1,11 +1,36 @@
 import clamd
-from flask import request
+from flask import current_app, request
+from psutil import net_connections, Process
 
 from . import status
 from ..clam import get_clamd_socket
 from dmutils.status import get_app_status, StatusError
 
 
+def get_clamd_status():
```

Comment:
A docstring here just for why we need it would help I think

Reply:
- [ ]